client/components/virtual-list/index.jsx

```diff
@@ -1,13 +1,20 @@
 import { AutoSizer, List } from '@automattic/react-virtualized';
 import classNames from 'classnames';
-import { localize } from 'i18n-calypso';
-import { debounce, range } from 'lodash';
+import { debounce } from 'lodash';
 import PropTypes from 'prop-types';
-import { cloneElement, Component } from 'react';
+import { cloneElement, createRef, Component } from 'react';
 
 const noop = () => {};
 
-export class VirtualList extends Component {
+function range( start, end ) {
```

Comment:
Is this something we could use in `js-utils`?

Reply:
Looks like a good idea. We have 12 usages of `range` in Calypso.

This implementation is a bit customized for the usage in this file. `range( 1, 3 )` includes the `end` value, i.e., produces `[ 1, 2, 3 ]` while the Lodash version produces `[ 1, 2 ]`. And it doesn't support the `step` param.